packages/stainless/src/stl.ts

```diff
@@ -162,6 +211,10 @@ type OpenAPIConfig = {
   spec: OpenAPIObject;
 };
 
+/**
+ * A type containing information about an API defined by the
+ * {@link Stl.api} method.
```

Comment:
I don't think comments like this add a lot of value, I prefer to just not include them myself

Reply:
Yeah, if a comment doesn't add new information it just distracts the reader and causes them to start ignoring more of the comments.